pkg/cmd/connector/connector.go

```diff
@@ -30,6 +31,7 @@ func NewConnectorsCommand(f *factory.Factory) *cobra.Command {
 	cmd.AddCommand(
 		cluster.NewConnectorClusterCommand(f),
 		namespace.NewNameSpaceCommand(f),
+		use.NewUseCommand(f),
```

Comment:
Use commands have been planned to be deprecated.
We have the same commands in context set-kafka etc.
It will be good to add set-connector command there.

My take is that we need 2 commands doing the same thing for now

Reply:
Yep I did see this and it was kinda weird ill add both to make it consistant until the change is made